alws/crud.py

```diff
@@ -782,3 +788,228 @@ async def commit_release(db: Session, release_id: int) -> (models.Release, str):
         selectinload(models.Release.platform)
     ))
     return release_res.scalars().first(), message
+
+
+async def get_repositories(db: Session, repository_id: int = None
+                           ) -> typing.List[models.Repository]:
+    repo_q = select(models.Repository)
+    if repository_id:
```

Comment:
The Zen of Python: Explicit is better than implicit.
So, `repository_id is not None` is better. And it helps to catch errors in the case when repository_id is zero or empty string.